.github/workflows/api.yml

```diff
@@ -0,0 +1,35 @@
+name: API
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./api
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - uses: actions/setup-node@v2
+      with:
+        node-version: '16.2.0'
```

Comment:
Could we avoid hardcoding versions in the CI process unless there is a valid reason for it, as mentioned in this discussion: https://github.com/mattermost/delivery-platform/pull/5#discussion_r1236750888